src/test/rustdoc-js/search_hidden.rs

```diff
@@ -0,0 +1,8 @@
+#![feature(doc_search_hidden)]
+
+#[doc(search_hidden)]
+/// Foo
+pub struct Bar;
```

Comment:
Can you also add a test that re-exports are still shown?
```suggestion
#[doc(search_hidden)]
/// Foo
pub struct Bar;

mod inner {
  pub use Bar;
}
```
and then update `search_hidden.js`.

Reply:
Sure!